src/kv/tx.h

```diff
@@ -239,6 +238,21 @@ namespace kv
       return version;
     }
 
+    /** Commit term if committed
+     *
+     * @return Commit term
+     */
+    Version commit_term()
```

Comment:
Minor: `get_commit_term()`?

Reply:
We have commit_version() above, so commit_term() I think.